src/backend/marsha/core/migrations/0073_siteconfig_is_logo_enabled.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.5 on 2023-10-06 13:59
```

Comment:
Maybe we should study squashing migrations... or maybe push it to 99 :sweat_smile:

Reply:
squashing them sounds a good solution.